MdeModulePkg/MdeModulePkg.dec

```diff
@@ -1593,6 +1593,10 @@
   # @Prompt Enable Capsule On Disk support.
   gEfiMdeModulePkgTokenSpaceGuid.PcdCapsuleOnDiskSupport|FALSE|BOOLEAN|0x0000002d
 
+  ## This PCD holds the background color of progress bar.
+  # @Prompt Describes background color of progress bar.

```

Comment:
Can you add a comment about byte order (i.e. Reserved, Red, Green, Blue)?

Reply:
Done.